### DISCUSSION TITLE
Huawei Band 6 not visible when connected to Phone

### DISCUSSION THREAD

**Discussion by abishur** [ESPresense/ESPresense#1107] [Q&A]
I have a Huawei Band 6 fitness watch.  I can see it when it's not connected to my phone, but when I turn on my phone's bluetooth (so I can see notifications from my phone) it stops broadcasting and I can't see it on fingerprint.

Obviously, the bluetooth is still on or else I wouldn't be able to get notifications from it anymore.  Is there a away to force it to keep broadcasting or to get the ESP32 to trying an connect to it even when it's not broadcasting?

**Comment by DTTerastar** [comment]
Some devices only broadcast for something to find it, but once found (connected) stop broadcasting.  It seems like that is the case for your Huawei Band 6.
